<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyObjectDialog</class>
 <widget class="QDialog" name="ModifyObjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>259</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Modify Object</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="tableLabel">
     <property name="text">
      <string>Select Table:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="tableComboBox"/>
   </item>
   <item>
    <widget class="QLabel" name="IDLabel">
     <property name="text">
      <string>Enter ID:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="IDEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="columnLabel">
     <property name="text">
      <string>Enter Column to Change:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="columnEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="valueLabel">
     <property name="text">
      <string>Enter New Value:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="valueEdit"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>IDEdit</zorder>
  <zorder>tableLabel</zorder>
  <zorder>tableComboBox</zorder>
  <zorder>columnLabel</zorder>
  <zorder>columnEdit</zorder>
  <zorder>buttonBox</zorder>
  <zorder>valueLabel</zorder>
  <zorder>valueEdit</zorder>
  <zorder>IDLabel</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ModifyObjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ModifyObjectDialog</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
